doc/reference/src/core-language.pdc

```diff
@@ -692,9 +694,62 @@ To-do: Lay out syntax-directed typechecker following McBride's paper.
 
 ## Examples
 
+### Dependent identity function
```

Comment:
This section can be removed as it is the same as the I combinator below?

Reply:
Aye, good catch, removed.